tes/utils.py

```diff
@@ -77,16 +83,8 @@ def _unmarshal(v: Any, obj: Type) -> Any:
         field = v
         omap = fullOmap.get(o.__name__, {})
         if k in omap:
-            if isinstance(omap[k], tuple):
-                try:
-                    obj = omap[k][0]
-                    field = _unmarshal(v, obj)
-                except Exception:
-                    obj = omap[k][1]
-                    field = _unmarshal(v, obj)
```

Comment:
I have unmarshalled to to every TES object and this code never ran. Also the model definitions do not contain any tuples. Maybe this was added to support some legacy model, but I'm pretty sure the code is not used with the current models.

Anyway, instead of trying to create some artificial class to use with the `unmarshal()` function to force this code to run, I have just removed it. We can always add more code if needed by future model versions.

Or maybe I missed something? Do you know why these tuple conditionals made it here in the first place?

Reply:
My guess is that the tuple conditionals were added when `outputs` and `logs` were represented as tuples in a much earlier TES schema (first added here in commit 8f43a45): 

https://github.com/ohsu-comp-bio/py-tes/blob/8f43a45b593f12096b1ce97a6c40a628e8622fea/tes/utils.py#L35-L43

Agreed on the removal as it's no longer necessary.